aiodns/__init__.py

```diff
@@ -6,7 +6,14 @@
     import trollius as asyncio
 import pycares
 
-from . import error
+from typing import (
+    Any,
+    List,
+    Optional,
+)
+
+# TODO: Work out mypy no attribute error
+from . import error # type: ignore
```

Comment:
Why is this import needed?

Reply:
@cooperlees What was the issue that you encountered? Seems to work fine with Python3 mypy, so I guess that this workaround should be removed.

Reply:
https://github.com/saghul/aiodns/pull/75